### PR TITLE
Add setup and configuration for reddit image resizer patch and refresh_token

### PR DESCRIPTION
**Pull request by rhysyngsun**
This patches reddit with a custom resizer that doesn't error on the callsite. It also adds our refresh_token plugin as well.

To test:

- `git clone` this repo, mitodl/refresh_token, and mitodl/no_op2_resizer alongside your reddit/reddit checkout.
- Rebuild your vagrant: `vagrant destroy -f && vagrant up`
- Verify you can hit the reddit.local site and see no errors in `/var/log/upstart/reddit-pastor.log`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,15 @@
...
-    vagrant up
+### Setup
+```bash
+cd ..
+git clone git@github.com:reddit/reddit.git
+git clone git@github.com:mitodl/reddit-config.git
+git clone git@github.com:mitodl/refresh_token.git
+git clone git@github.com:mitodl/no_op2_resizer.git
+cd reddit
+git apply patch ../reddit-config/patches/configure_tracing.patch
```
Comment (noisecapella) [inline README.md]: This should be `git apply` instead of `git apply patch`

File: NOTES.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Notes
+---
+
+- Reddit app will run if a plugin is missing
+- Reddit app will **not** run if a provider is missing
+- Reddit/Pylons uses `pkg_resources.WorkingSet` to load plugins/provider (see next note).
+- Plugins/providers need to be installed at the global level by invoking this:
```
Comment (noisecapella) [inline NOTES.md:9]: Who needs to do this step? Should these notes be moved into `README` for better visibility?

**Review by noisecapella — CHANGES_REQUESTED** [review]

File: NOTES.md (hunk 2 above)
Comment (rhysyngsun) [reply NOTES.md:9]: These are just general notes on findings when working with reddit, probably mostly going to be useful for the Ops team eventually.

Comment (noisecapella) [reply NOTES.md:9]: Can you a message that this is only relevant to the Ops team?

**Review by noisecapella — APPROVED** [review]

**Pull request merged**